src/components/CommentCard.jsx

```diff
@@ -0,0 +1,17 @@
+import "./styles.css";
```

Comment:
Though you have done a great job of adding comments, adding the comments to each article is a ticket later on, and as it's important to keep the functionality on each ticket separate - could you please remove the comment section from this ticket? 🙈